src/components/global-style/global-style.js

```diff
@@ -35,6 +35,14 @@ const GlobalStyle = createGlobalStyle`
     text-decoration: none;
     transition: color 0.2s ease-in-out;
   }
+
+  header a, footer a {
+    color: ${(props) => props.theme.colors.link.nav[0]};
+
+    &:hover {
+      color: ${(props) => props.theme.colors.link.nav[1]}
+    }
```

Comment:
From my perspective these styles are too specific to put into the global style. But that's just my view of 😄

Reply:
Agreed. I’ve been thinking about introducing a “shared” styled components file so styles can be shared across components without polluting global styles. I’ll look to introduce something later today.

I’ve also been chewing on how I’m going to employ a responsive strategy across the app. It’s been awhile since I’ve touched styled components, so I’m not used to the approaches these days. Anyway, that’s a different discussion entirely.

Reply:
I wouldn't overthink it tbh. It's just a handful of links in two components. So further abstracting that may lead to more confusion and overhead than it brings value.